src/version.h.in

```diff
@@ -26,4 +26,7 @@
 #define MIGRAPHX_VERSION_MINOR @PROJECT_VERSION_MINOR@
 #define MIGRAPHX_VERSION_PATCH @PROJECT_VERSION_PATCH@
 #define MIGRAPHX_VERSION_TWEAK @PROJECT_VERSION_TWEAK@
+#define MIGRAPHX_MIOPEN_VERSION_MAJOR @miopen_VERSION_MAJOR@
+#define MIGRAPHX_MIOPEN_VERSION_MINOR @miopen_VERSION_MINOR@
+#define MIGRAPHX_MIOPEN_VERSION_PATCH @miopen_VERSION_PATCH@
```

Comment:
Remove these defines, They dont make sense when not building gpu backend.

Reply:
Done.